pkgs/games/nxengine/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchurl
+, libdrm
+, SDL
+, SDL_ttf
+}:
+
+# Note that this only includes the game engine, not the game data.
+# For the game data, see the website.
+
+stdenv.mkDerivation rec {
+  name = "nxengine-${version}";
+  version = "1.0.0.6";
+  src = fetchurl {
+    url = http://nxengine.sourceforge.net/dl/nx-src-1006.tar.bz2;
```

Comment:
Please use ``mirror://sourceforge/``.

Reply:
I'm not sure how that would work.  This is on the "site" page, not the releases.  So, I don't think mirror works here?  Or if it does, how do I change it to be of that form?

Reply:
Oh you are right, I suspected the link would be to releases. Odd.